common/management/commands/create_test_data.py

```diff
@@ -0,0 +1,1775 @@
+    text_entering="Creating cancel reasons...",
+    text_exiting="Cancel reasons created!",
+)
+def _create_cancel_reasons(*, number: int = 10) -> list[ReservationCancelReason]:
+    cancel_reasons: list[ReservationCancelReason] = []
+    for i in range(number):
+        cancel_reason = ReservationCancelReason(
+            reason=f"Reservation Cancel Reason {i}",
+            reason_fi=f"Reservation Cancel Reason {i}",
+            reason_sv=f"Reservation Cancel Reason {i}",
+            reason_en=f"Reservation Cancel Reason {i}",
+        )
+        cancel_reasons.append(cancel_reason)
+
+    return ReservationCancelReason.objects.bulk_create(cancel_reasons)
+
+
+@with_logs(
+    text_entering="Creating deny reasons...",
+    text_exiting="Deny reasons created!",
+)
+def _create_deny_reasons(*, number: int = 10) -> list[ReservationDenyReason]:
+    deny_reasons: list[ReservationDenyReason] = []
+    for i in range(number):
+        deny_reason = ReservationDenyReason(
+            reason=f"Reservation Deny Reason {i}",
+            reason_fi=f"Reservation Deny Reason {i}",
+            reason_sv=f"Reservation Deny Reason {i}",
+            reason_en=f"Reservation Deny Reason {i}",
+        )
+        deny_reasons.append(deny_reason)
+
+    return ReservationDenyReason.objects.bulk_create(deny_reasons)
+
+
+@with_logs(
+    text_entering="Creating pricings...",
+    text_exiting="Pricings created!",
+)
+def _create_pricings(
+    reservation_units: list[ReservationUnit],
+) -> list[ReservationUnitPricing]:
+    tax_percentages = _create_tax_percentages()
+    zero_tax = next(tax for tax in tax_percentages if float(tax.value) == 0.0)
+
+    pricings: list[ReservationUnitPricing] = []
+    for reservation_unit in reservation_units:
+        pricing_type = random.choice(PricingType.values)
+
+        pricing = ReservationUnitPricing(
+            begins=date(2021, 1, 1),
+            pricing_type=pricing_type,
+            price_unit=random.choice(PriceUnit.values),
+            lowest_price=0,
+            lowest_price_net=0,
+            highest_price=0,
+            highest_price_net=0,
+            tax_percentage=zero_tax,
+            status=PricingStatus.PRICING_STATUS_ACTIVE.value,
+            reservation_unit=reservation_unit,
+        )
+        pricings.append(pricing)
+
+        # Around 1/5 of the reservation units have two pricings
+        add_another: bool = random.choices([True, False], weights=[1, 5])[0]
+        if not add_another:
+            continue
+
+        tax_percentage: TaxPercentage = random.choice(tax_percentages)
+        multiplier = Decimal(f"1.{int(tax_percentage.value)}")
+
+        lowest_net = random.randint(0, 100)
+        highest_net = random.randint(lowest_net, 1000)
+        lowest = float(lowest_net * multiplier)
+        highest = float(highest_net * multiplier)
+
+        # Around 1/10 of the pricings are inactive
+        pricing_status: str = random.choices(PricingStatus.values, weights=[1, 10, 0])
```

Comment:
**Pricings**: does the randomization sometimes produce two active pricings for the same reservation unit? In any case with overlapping time-period = same begins.

Reply:
Yes, there might be two active pricings. Is this an issue?

Reply:
Yeah there should only be one = the current on one particular res.unit (begins <= now). The rest are its past or future.

Reply:
Ok, thought that maybe there could be one active free and paid pricing. I'll simplify this bit then.

Reply:
Done.